pkg/cmd/pipeline/logs.go

```diff
@@ -193,7 +203,9 @@ func allPipelines(p cli.Params) ([]string, error) {
 	if err != nil {
 		return nil, err
 	}
-
+	if len(ps.Items) == 0 {
```

Comment:
in an interactive mode, why is this an error? if the user has *not* specified a `pipeline` then this shouldn't be an error.